etc/periodic/hourly/cleanup

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+ruby /usr/local/bin/cleanup.rb -l 95G /srv/backstore
```

Comment:
How does this limit work? Cleanup when the disk usage is above 95G?

Running once per hour that leaves a max of 5 new GiB's per hour if the shit hits the fan. Maybe lower it?

Reply:
Should the limit be percent based for when the disk is increased?